libs/designsystem/src/lib/components/action-sheet/action-sheet-popout/action-sheet-popout.component.spec.ts

```diff
@@ -19,35 +19,37 @@ describe('ActionSheetPopoutComponent', () => {
   let component: ActionSheetPopoutComponent;
   let fixture: ComponentFixture<ActionSheetPopoutComponent>;
 
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      declarations: [
-        ActionSheetPopoutComponent,
-        ListComponent,
-        CardComponent,
-        CardHeaderComponent,
-        GroupByPipe,
-        SpinnerComponent,
-        InfiniteScrollDirective,
-        ButtonComponent,
-        ListItemColorDirective,
-        MockComponent(ionic.IonList),
-        MockComponent(ionic.IonListHeader),
-        MockComponent(ionic.IonLabel),
-        MockComponent(ionic.IonItem),
-        MockComponent(ionic.IonItemDivider),
-        MockComponent(ionic.IonItemGroup),
-        MockComponent(ionic.IonItemOptions),
-        MockComponent(ionic.IonItemSliding),
-      ],
-      schemas: [CUSTOM_ELEMENTS_SCHEMA],
-    }).compileComponents();
-    TestBed.overrideModule(BrowserDynamicTestingModule, {
-      set: {
-        entryComponents: [ActionSheetPopoutComponent],
-      },
-    });
-  }));
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        declarations: [
+          ActionSheetPopoutComponent,
+          ListComponent,
+          CardComponent,
+          CardHeaderComponent,
+          GroupByPipe,
+          SpinnerComponent,
+          InfiniteScrollDirective,
+          ButtonComponent,
+          ListItemColorDirective,
+          MockComponent(ionic.IonList),
+          MockComponent(ionic.IonListHeader),
+          MockComponent(ionic.IonLabel),
+          MockComponent(ionic.IonItem),
+          MockComponent(ionic.IonItemDivider),
+          MockComponent(ionic.IonItemGroup),
+          MockComponent(ionic.IonItemOptions),
+          MockComponent(ionic.IonItemSliding),
```

Comment:
This seems rather bloated/unneeded? 🤔 
Also consider replacing with spectator instead....